src/LiveComponent/doc/index.rst

```diff
@@ -352,6 +352,12 @@ clicked). To do that, use ``norender`` modifier:
 
     <input data-model="norender|max">
 
+You might as well deref for all fields ``*`` of the form:
+
+.. code-block:: html+twig
+
+    {{ form_start(form, {attr: {'data-model': 'norender|*'}}) }}
```

Comment:
You can also override this method - https://github.com/symfony/ux/blob/2.x/src/LiveComponent/src/ComponentWithFormTrait.php#L226-L229 - that's how I imagined people would do this :)